lib/tapioca/gem/listeners/mixins.rb

```diff
@@ -98,7 +98,15 @@ def interesting_ancestors_of(constant)
           # inherited ancestors, past the location of the constant itself.
           ancestors = Set.new.compare_by_identity.merge(ancestors_of(constant))
 
-          (ancestors - inherited_ancestors).to_a
+          result = (ancestors - inherited_ancestors).to_a
+
+          # Filter out Kernel for modules (but not classes) since modules shouldn't
+          # need to explicitly include Kernel in RBIs
+          unless constant.is_a?(Class)
```

Comment:
Let's use:
```suggestion
          unless Class === constant
```
since `is_a?` may be different in `constant`.